backend/migrations/2023-06-24-125250_planting_timeline_data/up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE plantings ADD COLUMN add_date DATE;
+ALTER TABLE plantings ADD COLUMN remove_date DATE;
```

Comment:
Maybe `creation_date` and `deletion_date` is better?

Reply:
We have terminology about this, see 12glossary.md

It probably should be updated to full pairs, I suggest:

- add/remove (as this PR already uses) and
- create/delete

Reply:
I added those two fields, but I did not use them. 
The delete_date seems to be relevant only once we implement hard/soft delete.

Reply:
Yes. I tried to clarify the terms in https://github.com/ElektraInitiative/PermaplanT/pull/545/files#diff-94bf40d44642ef0b5cce441bf0ae1f48175578fee52f720a298e39119ebfc9e2